Sources/Subs.php

```diff
@@ -4852,9 +4852,6 @@ function inet_ptod($ip_address)
  */
 function inet_dtop($bin)
 {
-	if(strpos($bin,'.')!==false || strpos($bin,':')!==false)
-		return $bin;
```

Comment:
Then pg should use a new function. This one specifically mentions binary data and it is failing to use binary data correctly due to those checks.